.rubocop.yml

```diff
@@ -1,9 +1,7 @@
----
```

Comment:
Any particular reason why you're removing the document header?

Most YAML linters will complain mightily about the lack of one